internal/sessiontest/revocation_test.go

```diff
@@ -348,8 +349,8 @@ func TestRevocationAll(t *testing.T) {
 		sacctime := sacc.Accumulator.Time
 
 		// trigger time update and update accumulator
-		time.Sleep(time.Second)
```

Comment:
This line causes a timestamp to be updated, which is verified below using `require.NotEqual(t, sacctime, sacc.Accumulator.Time)`. The removal of this line causes this to fail, can you reinstate it?